pandas/tests/indexes/period/test_construction.py

```diff
@@ -320,19 +324,28 @@ def test_constructor_year_and_quarter(self):
     def test_constructor_freq_mult(self):
         # GH #7811
         for func in [PeriodIndex, period_range]:
-            # must be the same, but for sure...
-            pidx = func(start='2014-01', freq='2M', periods=4)
+
```

Comment:
Initial I was changing all the range-based PeriodIndex constructions in this file to `period_range`, but then I realized we should probably keep some as PeriodIndex, and just catch the warning. Let me know if you want me to change any back to PeriodIndex.

Reply:
Probably not going to have time to duplicate all the tests for period_range and PeriodIndex.

We have the basics covered for both, so I think things will be OK.

We have a test specifically for the deprecation, so that should be fine too.

Reply:
can you parameterize this, would be a lot more clearn